airbyte-integrations/connectors/destination-dev-null/src/main/kotlin/io/airbyte/integrations/destination/dev_null/TestingDestinations.kt

```diff
@@ -42,6 +48,33 @@ constructor(
         ]
     }
 
+    override fun spec(): ConnectorSpecification {
+        if (isCloudDeployment) {
+            return super.spec()
+        } else {
+            /** 1. Update the title. 2. Only keep the "silent" mode. */
+            val spec = super.spec()
+
+            (spec.connectionSpecification as ObjectNode).put("title", DEV_NULL_DESTINATION_TITLE)
```

Comment:
This is the cloud title also?

Reply:
I guess I fucked that one up? I reverted them. I'll fix

Reply:
actually, why should we use different names, now that they're the same connector?

Reply:
🤷 Minimal disruption? Keeping parity with what exists?